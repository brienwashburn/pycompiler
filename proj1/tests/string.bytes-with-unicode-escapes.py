b"\N{MIDDLE DOT}"
b"\U00000040"
b"\u0040"
