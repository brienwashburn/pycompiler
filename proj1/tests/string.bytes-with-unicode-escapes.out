(LIT #"\\N{MIDDLE DOT}")
(NEWLINE)
(LIT #"\\U00000040")
(NEWLINE)
(LIT #"\\u0040")
(NEWLINE)
(ENDMARKER)
